executor/windowfunc/window_funcs.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 PingCAP, Inc.
```

Comment:
Package windowfunc should be contained by expression instead of executor?

Reply:
I think it is better to be in executor because it only deals with the execution stage and it is uniform with package `aggfuncs`.